src/shared/locales/enUs/translations/labs/index.ts

```diff
@@ -4,6 +4,7 @@ export default {
     filterTitle: 'Filter by status',
     search: 'Search labs',
     successfullyUpdated: 'Successfully updated',
+    successfullyCompleted: 'Succesfully Completed',
```

Comment:
I believe @jackcmeyer  wanted this to be `Successfully completed ${type} for ${patient}`

Reply:
should i add 'for' text? Cause the original text must be `${t('labs.successfullyCompleted')} ${complete.type} ${patient?.fullName} ` @morrme @jackcmeyer